src/test/groovy/life/qbic/datamodel/dtos/business/CustomerSpec.groovy

```diff
@@ -36,10 +36,10 @@ class CustomerSpec extends Specification{
 
   def "missing affiliation list shall cause the customer to contain an empty list"() {
     when:
-    def customer = new Customer("", "", AcademicTitle.NONE, "a.b@c.de", null)
+    def projectManager = new ProjectManager("", "", AcademicTitle.NONE, "a.b@c.de", null)
     then:
-    customer.affiliations != null
-    customer.affiliations.isEmpty()
+    projectManager.affiliations != null
+    projectManager.affiliations.isEmpty()
```

Comment:
Since this is the customer spec I would keep the customer here.. if you dont want to keep the customer here I would suggest to update the comment :)

Reply:
I would keep the old test, as I see you created a ProjectManagerSpec so no need to include the project manager to the customer spec

Reply:
No one saw that... >< Nice Catch!